priv/templates/nimble_template/test/otp_app_web/views/api/error_view_test.exs

```diff
@@ -0,0 +1,96 @@
+defmodule <%= web_module %>.Api.ErrorViewTest do
+  use <%= web_module %>.ConnCase, async: true
+
+  # Bring render/3 and render_to_string/3 for testing custom views
+  import Phoenix.View
```

Comment:
should we use ViewCase here and then we can remove the import ✨ 
```suggestion
  use <%= web_module %>.ViewCase, async: true
```

Reply:
yup, i updated in https://github.com/nimblehq/elixir-templates/pull/140/commits/e355bb4b1264d6ff57a02f3ebf736394b52781d3